app/test_main.py

```diff
@@ -1,3 +1,27 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+class TestGetHumanAge:
+    @pytest.mark.parametrize("cat_age,dog_age,expected", [
+        pytest.param(0, 0, [0, 0],
+                     id="Should return zeros, when initial data is zero"),
+        pytest.param(14, 14, [0, 0],
+                     id="Should return zeros, when ages are less than 15"),
```

Comment:
The expected result `[0, 0]` for ages `14, 14` is correct according to the task description, as both cat and dog ages are less than 15.